packages/wrangler/src/__tests__/workflows.test.ts

```diff
@@ -480,6 +501,47 @@ describe("wrangler workflows", () => {
 		});
 	});
 
+	describe("instances send-event", () => {
+		const mockInstances: Instance[] = [
+			{
+				id: "foo",
+				created_on: mockCreateDate.toISOString(),
+				modified_on: mockModifiedDate.toISOString(),
+				workflow_id: "b",
+				version_id: "c",
+				status: "running",
+			},
+		];
+
+		const event = "my-event";
+
+		it("should send an event without payload to the bar instance given a name", async () => {
+			writeWranglerConfig();
+			await mockGetInstances(mockInstances);
+			await mockSendEventRequest("bar", event);
+
+			await runWrangler(
+				`workflows instances send-event some-workflow bar ${event}`
```

Comment:
I don't think using `${event}` really helps when `my-event` is used on the expectation right below?